app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
```

Comment:
That being said, I'm not entirely sure what would happen with NoMemoryError or something. That strikes me as a type of Exception you do not want to catch.